sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -155,8 +155,8 @@ public OfferEvaluator(
             description = "existing";
             shouldGetNewRequirement = false;
         }
-        logger.info("Generating requirement for {} pod '{}' containing tasks: {}",
-                description, podInstance.getName(), podInstanceRequirement.getTasksToLaunch());
+        logger.info("Generating requirement for {} pod '{}' containing tasks: {}, situation is {}",
```

Comment:
Something like  `Generating requirement for {} pod '{}' containing tasks: {}. This is a(n) {} launch` makes more sense to me.